src/hooks/createRequest.js

```diff
@@ -16,9 +16,9 @@ export default function createRequestObject (type, filter) {
         body: JSON.stringify(filter)
       }
     }
-  case 'browse_collection':
+  case 'browsecollection':
```

Comment:
why no underscore here? I feel like its better with the underscore

Reply:
everything else is no underscore so I thought it was more consistent

Reply:
but I guess we can add underscores

Reply:
actually I think it's better with no underscores because it matches the api endpoint names